src/helpers/performance-report-parser.helper.js

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs';
+import chalk from 'chalk';
+
+class PerformanceReportParse {
+  constructor() {
+    this.performanceReport = (fileName) => JSON.parse(fs.readFileSync(`reports/performance/${fileName}`, 'utf8'));
```

Comment:
1. This method should be declared outside constructor
2. It's be better to call it `getReport` or similar

Reply:
Changed.